portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java

```diff
@@ -128,6 +128,10 @@ public static LocaleUtil getInstance() {
 		return _localeUtil._getISOLanguages(locale);
 	}
 
+	public static String getLocaleDisplayName(Locale curLocale, Locale locale) {
```

Comment:
Why not rename "curLocale" to "displayLocale" as in the private method?

Reply:
perfect, thanks for pointing at it

 ![](https://media0.giphy.com/media/RUK8UQS11WlFtQgWhs/giphy.gif)